### PR TITLE
Skip comments in repository mapping file

### PR DESCRIPTION
**Pull request by vojtechsokol**

### CODE REVIEW

**Comment by Rezney** [comment]
Looks good to me. 

As per https://stackoverflow.com/questions/1961006/can-a-csv-file-have-a-comment it looks like there is no standard about how comments should be handled in csv so we might add one comment to our repomap file to indicate that it is indeed "#" in our case?

File: repos/system_upgrade/el7toel8/actors/repositoriesmapping/libraries/library.py [hunk 1]
```diff
@@ -15,6 +15,13 @@ def inhibit_upgrade(title):
     raise StopActorExecution()
 
 
+def strip_comments(file_):
+    for line in file_:
+        data = line.split('#')[0].strip()
```
Comment (Rezney) [inline repos/system_upgrade/el7toel8/actors/repositoriesmapping/libraries/library.py]: What about if we ignore a line starting with "#" and communicate that only new line comments are valid?

**Comment by vojtechsokol** [comment]
@Rezney There are comments in the repository mapping file that is updated for alt arches and.

File: repos/system_upgrade/el7toel8/actors/repositoriesmapping/libraries/library.py [hunk 2]
```diff
@@ -28,10 +28,15 @@ def scan_repositories(path):
         next(data)  # skip header
 
         for row in data:
-            if len(row) != 6:
+            # skip empty lines and comments
+            if not row or row[0].startswith('#'):
```
Comment (Rezney) [inline repos/system_upgrade/el7toel8/actors/repositoriesmapping/libraries/library.py]: I guess we will not catch empty lines if they begin with white-space, however not sure whether we care that much as those will be caught later as invalid record...

Comment (vojtechsokol) [reply repos/system_upgrade/el7toel8/actors/repositoriesmapping/libraries/library.py]: That is on purpose, I think it is better to keep the parsing as simple as possible, which servers two purposes: 1) less complex code, 2) motivation to keep the repomapping file clean.

Comment (Rezney) [reply repos/system_upgrade/el7toel8/actors/repositoriesmapping/libraries/library.py]: fair enough :)

**Pull request merged**